api/envoy/extensions/geoip_providers/maxmind/v3/maxmind.proto

```diff
@@ -48,6 +49,13 @@ message MaxMindConfig {
   // ``isp_db`` instead.
   string isp_db_path = 5 [(validate.rules).string = {pattern: "^$|^.*\\.mmdb$"}];
 
+  // Full file path to the MaxMind Country database, e.g., ``/etc/GeoLite2-Country.mmdb``.
+  // Database file is expected to have ``.mmdb`` extension.
+  //
+  // If ``country_db_path`` is not defined, country information will be fetched from
```

Comment:
if country_bd is not specified, country information will be fetched from city db if city_db is configured (if i still remember the implementation code).

Reply:
Will refactor it to clarify.